wiki/Tournaments/MWC/2022_7K/en.md

```diff
@@ -93,8 +93,51 @@ The osu!mania 7K 2022 is run by the osu! team and various community members.
 | ![][flag_GB] | **United Kingdom** | **[Vygatron](https://osu.ppy.sh/users/3628783)**, [username1947](https://osu.ppy.sh/users/16162078), [Lelloq](https://osu.ppy.sh/users/8610776), [aid\_](https://osu.ppy.sh/users/12808079) |
 | ![][flag_US] | **United States** | **[Alter-](https://osu.ppy.sh/users/4980256)**, [Terni](https://osu.ppy.sh/users/3279570), [\[GS\]Teo](https://osu.ppy.sh/users/7081478), [RevVoJH](https://osu.ppy.sh/users/7286896) |
 
+## Match schedule: Round of 16
+
+### Saturday, January 22, 2022
+
+| Team A | | | Team B | Match time | Local time A | Local time B |
+| --: | --: | :-- | :-- | :-: | :-: | :-: |
+| South Korea | ![][flag_KR] | ![][flag_CH] | Switzerland | Jan 22 (Sat) 11:00 UTC | Jan 22 (Sat) 20:00 UTC+9 | Jan 22 (Sat) 12:00 UTC+1 |
+| Malaysia | ![][flag_MY] | ![][flag_US] | United States | Jan 22 (Sat) 12:00 UTC | Jan 22 (Sat) 20:00 UTC+8 | Jan 22 (Sat) 08:00 UTC-4 |
+| Indonesia | ![][flag_ID] | ![][flag_CL] | Chile | Jan 22 (Sat) 13:00 UTC | Jan 22 (Sat) 20:00 UTC+7 | Jan 22 (Sat) 10:00 UTC-3 |
+| Spain | ![][flag_ES] | ![][flag_JP] | Japan | Jan 22 (Sat) 14:00 UTC | Jan 22 (Sat) 15:00 UTC+1 | Jan 22 (Sat) 23:00 UTC+9 |
+
+### Sunday, January 23, 2022
+
+| Team A | | | Team B | Match time | Local time A | Local time B |
+| --: | --: | :-- | :-- | :-: | :-: | :-: |
+| China | ![][flag_CN] | ![][flag_TW] | Taiwan | Jan 23 (Sun) 08:00 UTC | Jan 23 (Sun) 16:00 UTC+8 | Jan 23 (Sun) 16:00 UTC+8 |
+| Thailand | ![][flag_TH] | ![][flag_RU] | Russian Federation | Jan 23 (Sun) 11:00 UTC | Jan 23 (Sun) 18:00 UTC+7 | Jan 23 (Sun) 14:00 UTC+3 |
+| Philippines | ![][flag_PH] | ![][flag_PL] | Poland | Jan 23 (Sun) 12:00 UTC | Jan 23 (Sun) 20:00 UTC+8 | Jan 23 (Sun) 13:00 UTC+1 |
+| Canada | ![][flag_CA] | ![][flag_BR] | Brazil | Jan 23 (Sun) 20:00 UTC | Jan 23 (Sun) 15:00 UTC-5 | Jan 23 (Sun) 17:00 UTC-3 |
+
 ## Mappools
 
+### Round of 16
+
+**[Download the mappack here (89 MB)](https://drive.google.com/uc?id=1uc1oILMX9J7dw1huWo3Kd3r31qROfIRm)**
+
+- Rice
+  1. [Laur - Attractor Dimension (Wonki) \[7K\] Insane\]](https://osu.ppy.sh/beatmapsets/1587984#mania/3243265)
```

Comment:
the other ones have similar mistakes

```suggestion
  1. [Laur - Attractor Dimension (Wonki) \[\[7K\] Insane\]](https://osu.ppy.sh/beatmapsets/1587984#mania/3243265)
```

Reply:
looks like my regex rule did not account for nested brackets

Reply:
nvm looks like bazzadward noticed and i never got the refresh prompt